instrumentation/opentelemetry-instrumentation-pyramid/src/opentelemetry/instrumentation/pyramid/callbacks.py

```diff
@@ -180,7 +193,8 @@ def trace_tween(request):
                 else:
                     activation.__exit__(None, None, None)
 
-                context.detach(request.environ.get(_ENVIRON_TOKEN))
+                if request.environ.get(_ENVIRON_TOKEN, None) is not None:
+                    context.detach(request.environ.get(_ENVIRON_TOKEN))
```

Comment:
nit: unnecessary double dictionary lookup here. Could just grab the token in a var and use that.

Reply:
Done